lib/color.zsh

```diff
@@ -0,0 +1,20 @@
+# Define how to colorize before the variables
+prompt_geometry_colorize() {
```

Comment:
Not sure. `color_colorize` doesn't read well to me. Since we keep using this function throughout the prompt why not a shorthand name? Even if it's `-g-color`, just to save characters.